config.json

```diff
@@ -22,48 +22,47 @@
       ]
     },
     {
-      "slug": "leap",
-      "uuid": "66d974b5-18fc-4993-b5f2-7beda4f4afa3",
+      "slug": "two-fer",
+      "uuid": "2ee3cc7a-db3f-4668-9983-ed6d0fea95d1",
       "core": true,
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-        "control_flow_if_else_statements",
-        "integers"
+        "optional_values",
+        "strings"
       ]
     },
     {
-      "slug": "bob",
-      "uuid": "b3c4d578-10c8-47bc-b0ae-149ed8da530a",
+      "slug": "leap",
```

Comment:
It will be really interesting to see what happens to leap's continuation rate (hopefully it increases). Let's keep monitoring this.